lib/hanami/logger.rb

```diff
@@ -142,7 +146,7 @@ def _message_hash(message)
       # @since x.x.x
       # @api private
       def _format(hash)
-        hash.map { |k, v| "#{k}=#{v}" }.join(SEPARATOR)
+        hash.map { |k, v| "#{k}=#{v}" unless v.nil? }.join(SEPARATOR) + NEW_LINE
```

Comment:
I am 👍 for keeping empty value. It does take a bit more spaces but in return the format stays consistent for log consumer to play with.